### PR TITLE
Renaming

### PR DESCRIPTION
**Pull request by silverdaz**
I renamed some components.

### Describe the pull request:

- [ ] Bug fix
- [x] Functional change
- [ ] New feature
- [ ] Code cleanup
- [x] Build system change
- [ ] Documentation change
- [ ] Language translation

### CODE REVIEW

**Comment by omllobet** [comment]
The EGA-archive repo is now up to date with the NBIS repo.
The work can continue on this repo, without affecting the current deployments (targeting the NBIS repo).

The tests are the same as the ones in the NBIS repo (except that we don't build the nbisweden/ega-* images, we only pull them now).

Please review this when you can.  Ping Joakim @jbygdell and Johan @viklund

File: README.md [hunk 1]
```diff
@@ -26,7 +26,8 @@ LocalEGA is divided into several components, as docker containers.
 | ingesters   | Split the Crypt4GH header and move the remainder to the storage backend. No cryptographic task, nor access to the decryption keys. |
 | verifiers   | Decrypt the stored files and checksum them against their embedded checksum. |
 | vault       | Storage backend: as a regular file system or as a S3 object store. |
-| outgesters  | Front-facing check for download permissions. |
+| finalizers  | Handle the so-called _Stable ID_ filename mappings from CentralEGA. |
```
Comment (jbygdell) [inline README.md]: This part can be merged with the other changes to the readme file

File: deploy/bootstrap/lega.sh [hunk 2]
```diff
@@ -84,7 +84,7 @@ chroot_sessions = True
 
 [vault]
 driver = S3Storage
-url = http://s3:9000
+url = http://vault:9000
```
Comment (jbygdell) [inline deploy/bootstrap/lega.sh]: The s3 -> vault changes should be placed into a separate commit.

File: deploy/bootstrap/lega.sh [hunk 3]
```diff
@@ -248,7 +248,7 @@ cat >> ${PRIVATE}/lega.yml <<EOF
     restart: on-failure:3
     networks:
       - lega
-    entrypoint: ["gosu", "lega", "ega-finalize"]
+    entrypoint: ["gosu", "lega", "ega-id-mapper"]
```
Comment (jbygdell) [inline deploy/bootstrap/lega.sh]: What's the point of renaming id-mapper to finalize when you have to do a ugly hack to get the tests to pass?

Comment (silverdaz) [reply deploy/bootstrap/lega.sh]: The modules are up-to-date, but simply because we didn't re-build the image, the ega-finalize does not exist in the nbisweden/ega-base image (but ega-id-mapper does).

The tests do not cover the ega-finalize/id-mapper.
However, when we call it ega-finalize, it of course crashes. Combine that with the fact that the tests do count the number of components up, when one is down, none of the tests run.

We thought the simplest way was to keep the name ega-id-mapper, rather than updating the tests to avoid this situation. We'll tear them down in the next commit anyway.

File: README.md [hunk 4]
```diff
@@ -1,32 +1,33 @@
-# NBIS repository for the Local EGA project
+# Local EGA main repository
 
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/3dd83b28ec2041889bfb13641da76c5b)](https://www.codacy.com/app/NBIS/LocalEGA?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=NBISweden/LocalEGA&amp;utm_campaign=Badge_Grade)
-[![Build Status](https://travis-ci.org/NBISweden/LocalEGA.svg?branch=dev)](https://travis-ci.org/NBISweden/LocalEGA)
-[![Coverage Status](https://coveralls.io/repos/github/NBISweden/LocalEGA/badge.svg?branch=dev)](https://coveralls.io/github/NBISweden/LocalEGA?branch=dev)
+[![Build Status](https://travis-ci.org/EGA-archive/LocalEGA.svg?branch=dev)](https://travis-ci.org/EGA-archive/LocalEGA)
+[![Documentation Status](https://readthedocs.org/projects/localega/badge/?version=latest)](https://localega.readthedocs.io/en/latest/?badge=latest)
```
[the remark this answers is not in the document]
Comment (silverdaz) [reply README.md]: Obviously not. The badges need to point to the right repo and be renamed!

File: README.md (hunk 1 above)
Comment (silverdaz) [reply README.md]: maybe so, but that'll do it too

File: deploy/bootstrap/lega.sh (hunk 2 above)
Comment (silverdaz) [reply deploy/bootstrap/lega.sh]: Maybe so, but that'll do it too

File: .travis.yml [hunk 5]
```diff
@@ -15,8 +15,6 @@ services: docker
 stages:
   - name: unit tests
     if: type IN (push, pull_request)
-  - name: image build
```
[the remark this answers is not in the document]
Comment (silverdaz) [reply .travis.yml]: Obviously, we don't want to build images that'll be pushed to nbis, so it is removed. In another commit? ok, maybe so, but that'll do it too.

File: deploy/bootstrap/lega.sh (hunk 3 above)
Comment (silverdaz) [reply deploy/bootstrap/lega.sh]: Marking the conversation as resolved, to move forward on the todo list, as this is not a blocker

**Comment by silverdaz** [comment]
The commit cleanup is now in place, I hope it'll be ok, for it to be merged at the Sprint Review.
